### PR TITLE
[Tests] Fix some test file paths

### PR DESCRIPTION
**Pull request by phansys**
## Subject

Fix some test file paths.

I am targeting this branch, because these changes are BC.

### CODE REVIEW

File: tests/Block/Service/AbstractBlockServiceTestCase.php [hunk 1]
```diff
@@ -11,14 +11,15 @@
  * file that was distributed with this source code.
  */
 
-namespace Sonata\BlockBundle\Test;
+namespace Sonata\BlockBundle\Tests\Block\Service;
```
Comment (kunicmarko20) [inline tests/Block/Service/AbstractBlockServiceTestCase.php]: I think the idea here is to make it reusable for other libraries and that is why it is in src and not tests

**Comment by core23** [comment]
This is not a bug, this is intended to reuse the classes in other projects

**Comment by phansys** [comment]
I understand, although I think it's a non common or standardised practice.
Is this intention focused to CI purposes in order to allow other packages of this project to leverage these classes, or to any project?

**Comment by kunicmarko20** [comment]
Its common practice, https://github.com/symfony/symfony/tree/4.4/src/Symfony/Bundle/FrameworkBundle/Test

**Comment by phansys** [comment]
Well, this seems reasonable to me then. I didn't see https://sonata-project.org/bundles/block/master/doc/reference/testing.html#test-blocks before.
Thank you.

**Pull request closed without merging**